load-test/send_load.sh

```diff
@@ -26,11 +26,32 @@ RPC="http://${IP}:${PORT}"
 acc1=$($DAEMON keys show account$FROM -a --home $DAEMON_HOME-1 --keyring-backend test)
 acc2=$($DAEMON keys show account$TO -a --home $DAEMON_HOME-1 --keyring-backend test)
 seq1=$("${DAEMON}" q account "${acc1}" --node $RPC --output json)
+seq2=$("${DAEMON}" q account "${acc2}" --node $RPC --output json)
 seq1no=$(echo "${seq1}" | jq -r '.sequence')
-bound=`expr 10000 + $seq1no`
-for (( a=$seq1no; a<$bound; a++ ))
+seq2no=$(echo "${seq2}" | jq -r '.sequence')
+balance1=$("${DAEMON}" q bank balances "${acc1}" --node $RPC --output json)
+balance1res=$(echo "${balance1}" | jq -r '.balances')
+echo "** Balance of Account 1 before send_load :: $balance1res **"
+balance2=$("${DAEMON}" q bank balances "${acc2}" --node $RPC --output json)
+balance2res=$(echo "${balance1}" | jq -r '.balances')
+echo "** Balance of Account 2 before send_load :: $balance2res **"
+bound1=`expr 10000 + $seq1no`
+bound2=`expr 10000 + $seq2no`
+for (( a=$seq1no; a<$bound1; a++ ))
 do
-    sTx=$("${DAEMON}" tx bank send "${acc1}" "${acc2}" 1000000"${DENOM}" --chain-id "${CHAINID}" --keyring-backend test --home $DAEMON_HOME-1 --node $RPC --output json -y --sequence $a) 
-    sTxHash=$(echo "${sTx}" | jq -r '.txhash')
-    echo "** TX HASH :: $sTxHash **"
+    sTx=$("${DAEMON}" tx bank send "${acc1}" "${acc2}" 1000000"${DENOM}" --chain-id "${CHAINID}" --keyring-backend test --home $DAEMON_HOME-1 --node $RPC --output json -y --sequence $a &) 
+    sTxHash=$(echo "${sTx}")
 done
+
+for (( b=$seq2no; b<$bound2; b++ ))
+do
+    sTx=$("${DAEMON}" tx bank send "${acc2}" "${acc1}" 1000000"${DENOM}" --chain-id "${CHAINID}" --keyring-backend test --home $DAEMON_HOME-1 --node $RPC --output json -y --sequence $b &) 
```

Comment:
I would prefer to merge the two loops:
```
for ...
   tx bank send acc1 acc2 amount
   tx bank send acc2 acc1 amount
done
```

Reply:
Combining the for loops will not work if the sequence of acc1 and acc2 is different as the sequence has to be incremented manually. All the txs will become invalid.

Reply:
how about this (pseudocode):
```
seq1 = get acc1 seq
seq2 = get acc2 seq
for i=0; i<limit; i++
   tx bank send acc1 acc2 ...  --sequence {seq1+i}
   tx bank send acc2 acc1 ...  --sequence {seq2+i} 
```